.devcontainer/ESP32/Dockerfile.ESP32.SRC

```
@@ -107,5 +109,21 @@ RUN apt-get autoremove -y \
     && apt-get clean -y \
     && rm -rf /var/lib/apt/lists/*
 
+# Create the non-root user and open up read/execute access to the ESP-IDF
+# toolchain and repositories that were downloaded/cloned as root above.
+RUN groupadd --gid $USER_GID $USERNAME \
```

Hadolint (2.14.0)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)